app/src/main/java/co/condorlabs/customcomponents/customcollapsibleview/CollapsibleView.kt

```diff
@@ -0,0 +1,585 @@
+package co.condorlabs.customcomponents.customcollapsibleview
```

Comment:
rename the methods under this file, most of them have wrong names, ask to your team mates if the names are meaningful

Reply:
Done